### PR TITLE
Update document-store-example.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Updated examples, the last one has changed a bit. In the actual docs page, this is using a different random collection. From my point of view if the example is within a site following a structured or sequential order of operations I think changing the collection’s name and going back 4 or 5 steps back to reproduce the same operations is not very instructive. I think it’s better to continue with operations following the same schema that the reader has been following, in this case with the employees collection.

### CODE REVIEW

File: docs/collections/documents/document-store-example.md [hunk 1]
```diff
@@ -86,16 +86,19 @@ client = C8Client(protocol='https', host=HOST, port=443,
...
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 2]
```diff
@@ -131,30 +134,48 @@ print(client.get_fabric_details())
...
-        await console.log("result is: ", result);
-      } catch(e){
-        await console.log("Fabric details could not be fetched because "+ e);
-      }
-  }
-
-  getFabric();
+const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 3]
```diff
@@ -131,30 +134,48 @@ print(client.get_fabric_details())
...
+const fabric = "_system";
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 4]
```diff
@@ -192,32 +213,60 @@ client.create_collection(name=COLLECTION_NAME, stream=True)
...
-      return "Collection creation did not succeed due to " + e;
-    }
-
-    return "Collection " + collectionDetails.name + " created successfully";  
-  }
-
-  createCollection().then(console.log);
+const jsC8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 5]
```diff
@@ -192,32 +213,60 @@ client.create_collection(name=COLLECTION_NAME, stream=True)
...
+const collectionName = "employees";
+const client = new jsC8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 6]
```diff
@@ -254,32 +303,58 @@ client.add_hash_index(COLLECTION_NAME, fields=FIELDS, unique=False)
...
-      return "Index creation did not succeed due to " + e;
-    }
-
-    return "Index created successfully";  
-  }
-
-  createIndex().then(console.log);
+const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 7]
```diff
@@ -254,32 +303,58 @@ client.add_hash_index(COLLECTION_NAME, fields=FIELDS, unique=False)
...
+const collectionName = "employees";
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 8]
```diff
@@ -321,43 +396,82 @@ client.insert_document(collection_name=COLLECTION_NAME, document=DOCS)
...
-  async function populate() {
-    await console.log("Creating the collection object to be used and populating with documents...");
-    await client.insertDocumentMany("employees", docs);
-    await console.log("collection populated with documents");
-  }
-
-  populate();
+const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 9]
```diff
@@ -321,43 +396,82 @@ client.insert_document(collection_name=COLLECTION_NAME, document=DOCS)
...
+const collectionName = "employees";
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 10]
```diff
@@ -321,43 +396,82 @@ client.insert_document(collection_name=COLLECTION_NAME, document=DOCS)
...
+  };
+  console.log(message);
+}
+
+async function insertDataInCollection () {
+  await client
+    .login(email, password)
+    .then((e) => console.log("1. User authentication done!"))
+    .catch((error) => messageHandler(error));
+  console.log("2. Insert documents in collection " + collectionName);
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
  console.log("2. Insert documents into collection " + collectionName);
```

File: docs/collections/documents/document-store-example.md [hunk 11]
```diff
@@ -321,43 +396,82 @@ client.insert_document(collection_name=COLLECTION_NAME, document=DOCS)
...
+  await client
+    .insertDocumentMany(collectionName, docs)
+    .then((documentsDetails) => {
+      console.log(documentsDetails);
+      console.log("Documents inserted in collection " + collectionName);
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
      console.log("Documents inserted into collection " + collectionName);
```

File: docs/collections/documents/document-store-example.md [hunk 12]
```diff
@@ -395,27 +509,49 @@ print(f"Response from Query: {docs}")
...
-    const result = await client.executeQuery(
-      "FOR employee IN employees RETURN employee"
-    );
-    await console.log(result);
-  }
-
-  c8Queries();
+const jsC8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 13]
```diff
@@ -395,27 +509,49 @@ print(f"Response from Query: {docs}")
...
+const query = "FOR employee IN employees RETURN employee";
+const client = new jsC8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 14]
```diff
@@ -453,33 +589,58 @@ client.on_change(COLLECTION_NAME, callback=callback_fn)
...
-
-  async function realTimeListener() {
-    const listener = await client.onCollectionChange("employees");
-
-    listener.on('message',(msg) => console.log("message=>", msg));
-    listener.on('open',() => {
-        this.callback_fn(collection);
+const jsC8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 15]
```diff
@@ -453,33 +589,58 @@ client.on_change(COLLECTION_NAME, callback=callback_fn)
...
+const collectionName = "employees";
+const client = new jsC8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric,
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 16]
```diff
@@ -578,89 +739,139 @@ if __name__ == '__main__':
...
-
-    console.log(count.result);
-
-    await client.executeRestql("deleteData");
-  }
-
-  restqldemo().then(console.log("Starting Execution"));
+ const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/collections/documents/document-store-example.md [hunk 17]
```diff
@@ -578,89 +739,139 @@ if __name__ == '__main__':
...
+const fabric = "_system";
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
// Create an authenticated instance with an API key.
```

File: docs/collections/documents/document-store-example.md [hunk 18]
```diff
@@ -578,89 +739,139 @@ if __name__ == '__main__':
...
+    )
+    .catch((error) => messageHandler(error));
+  await client
+    .createRestql("getCount", getCount, {})
+    .then((restQl) =>
+      console.log("6. " + restQl.result.name + "query created.")
+    )
+    .catch((error) => messageHandler(error));
+
+  console.log("\n------- EXECUTING THE QUERIES  ------");
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
  console.log("\n------- RUNNING THE QUERIES  ------");
```

File: docs/collections/documents/document-store-example.md [hunk 19]
```diff
@@ -578,89 +739,139 @@ if __name__ == '__main__':
...
+
+  await client
+    .executeRestql("updateData", { lastname: "mathews" })
+    .then(() => console.log("\n9. Data updated."))
+    .catch((error) => messageHandler(error));
+
+  await client
+    .executeRestql("getData")
+    .then((res) => {
+      console.log("\n10. Output of get data query after update: ");
```
Comment (jamesm-macrometa) [inline docs/collections/documents/document-store-example.md]:
```suggestion
      console.log("\n10. Updated getData query output: ");
```

@diana-macrometa This one can probably be improved, any thoughts?

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!

File: docs/collections/documents/document-store-example.md (hunk 19 above)
Comment (diana-macrometa) [reply docs/collections/documents/document-store-example.md]: That's a toughie. The type of query is getData, so maybe use the proper name in the log to make clearer what is happening? If I'm reading this right, this sentence is right before the posting of the output of said query. Maybe:

Updated getData query output:

Comment (jamesm-macrometa) [reply docs/collections/documents/document-store-example.md]: Good idea. Thanks!

**Comment by diana-macrometa** [comment]
@edgargarciamacrometa, please respond to @jamesm-macrometa's edits so we can merge this.

**Comment by edgargarciamacrometa** [comment]
@diana-macrometa changes were committed

**Pull request merged**